test/sql/008_istore_key_gin_test.sql

```diff
@@ -1,128 +1,88 @@
-BEGIN;
 -- bigistore gin access should do index scan using gin;
--- ./spec/istore/istore_key_gin_spec.rb:23;
-CREATE EXTENSION istore;
-CREATE TABLE t AS
+CREATE TABLE t1 AS
 SELECT i, string_agg(j::text || '=>0', ',')::bigistore AS s
 FROM
-generate_series(0, 100000) AS i,
+generate_series(0, 10000) AS i,
 generate_series(i, i + (i % 10)) AS j
 GROUP BY i;
-;
-CREATE INDEX ON t USING gin(s);;
+CREATE INDEX ON t1 USING gin(s);;
 SET enable_seqscan = 0;;
-EXPLAIN (COSTS OFF) SELECT * FROM t WHERE s ? 3;;
+EXPLAIN (COSTS OFF) SELECT * FROM t1 WHERE s ? 3;;
 RESET enable_seqscan;;
-ROLLBACK;
-BEGIN;
 -- bigistore gin access should find the matching rows for key 300;
--- ./spec/istore/istore_key_gin_spec.rb:31;
-CREATE EXTENSION istore;
-CREATE TABLE t AS
+CREATE TABLE t2 AS
```

Comment:
Just wondering: now that we don't rollback after each test, does it make sense to create a new table with the same content each time? Couldn't we use `t1` for follow-up tests searching for keys `300`, `600` and `900`?
Same goes for table `t5` and following tests.